rollup-interface/src/state_machine/da.rs

```diff
@@ -212,8 +212,12 @@ pub struct Time {
     nanos: u32,
 }
 
-#[derive(Debug, Error)]
-#[error("Only intervals less than one second may be represented as nanoseconds")]
+#[derive(Debug)]
+#[cfg_attr(
+    feature = "std",
+    derive(thiserror::Error),
+    error("Only intervals less than one second may be represented as nanoseconds")
+)]
```

Comment:
Since implementing `core::error::Error` for this type seems very simple, we can easily do this even without `std` and `thiserror` so that the `no_std`/`std` APIs are more similar.

Reply:
Yes, indeed!

Reply:
Wdyt if we leave this change to this issue? https://github.com/Sovereign-Labs/sovereign-sdk/issues/1105

That way, we keep the changeset concise

Reply:
Works for me!